src/app/components/FetchComment.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../assets/styles/login.scss';
+import '../../assets/styles/fetchcomment.scss';
+import '../containers/CommentContainer';
```

Comment:
Is there a reason why you've imported the whole file and not assigned it to a variable? It also seems like this isn't being used 🤷‍♀️

Reply:
fixed it